src/__mocks__/axios.js

```diff
@@ -1,4 +1 @@
-const axios = require('axios')
-
-module.exports = axios
-module.exports.post = jest.fn(() => Promise.resolve())
+jest.mock('axios', () => jest.fn(async () => Promise.resolve()))
```

Comment:
Not needed for merging this is just FYI, here you could go even simpler with 

```suggestion
jest.mock('axios')
```

And there's probably an even better way to achieve what we're trying to do here in the jest config but couldn't find it in 5 minutes.

This is totally good enough 👍